tools/cpp_build/torchjit/torchjit_test.cpp

```diff
@@ -0,0 +1,14 @@
+#include "torch/csrc/jit/graph_executor.h"
+#include "test_jit.h"
+
+using namespace torch;
+
+// TODO: script
+
+int main(int argc, char* argv[])
+{
+  std::string out = jit::runJITCPPTests();
+  std::cout<<out<<std::endl;
+
+  return 0;
+}
```

Comment:
Can we remove the tests entirely for now? They're not even running in the CI. We should figure out a way to refactor our C++ test framework, and then we can add nice bindings here

Reply:
Done